build/docker/docker-compose.yml

```diff
@@ -32,6 +40,34 @@ services:
         aliases:
           - elasticsearch.sfs.local
 
+  kibana:
+    image: docker.elastic.co/kibana/kibana:7.9.2
+    container_name: kibana
+    environment:
+      SERVER_HOST: "0.0.0.0"
+      SERVER_PORT: 5601
+      SERVER_NAME: kibana
+      ELASTICSEARCH_HOSTS: http://elasticsearch.sf.local:9200
+    ports:
+      - "5601:5601"
+    depends_on:
+      - elasticsearch
+    networks:
+      default:
+        aliases:
+          - kibana.sf.local
+
+  db-dos:
+    image: 461183108257.dkr.ecr.eu-west-2.amazonaws.com/uec-dos-api/tdb/database:latest
+    hostname: db-dos
+    container_name: db-dos
+    ports:
+      - 5432:5432
+    networks:
+      default:
+        aliases:
+          - dbdos.sf.local
```

Comment:
Can we change it to `db-dos.sfs.local`? 1) to match the container name 2) to reflect the project short name

Reply:
Done.